Html/Home/style.css

```diff
@@ -0,0 +1,195 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+* {
+  box-sizing: border-box;
```

Comment:
Sugiro dar uma olhada em "Reset CSS", para facilitar e padronizar.
Referências:
- https://meyerweb.com/eric/tools/css/reset/
- https://github.com/elad2412/the-new-css-reset
- https://github.com/necolas/normalize.css

Reply:
feito